hummingbot/templates/conf_global_TEMPLATE.yml

```diff
@@ -127,6 +127,9 @@ bybit_perpetual_secret_key: null
 bybit_perpetual_testnet_api_key: null
 bybit_perpetual_testnet_secret_key: null
 
+wazirx_api_key: null
+wazirx_secret_key: null
+
```

Comment:
When a template file is changed, the `template_version` should be changed too

Reply:
Thanks, updated.